docker/vttestserver/Dockerfile.mysql57

```diff
@@ -55,4 +55,4 @@ COPY --from=builder --chown=vitess:vitess /vt/install /vt
 VOLUME /vt/vtdataroot
 USER vitess
 
-CMD /vt/bin/vttestserver -port $PORT -keyspaces $KEYSPACES -num_shards $NUM_SHARDS -mysql_bind_host ${MYSQL_BIND_HOST:-127.0.0.1}
+CMD /vt/bin/vttestserver -port $PORT -keyspaces $KEYSPACES -num_shards $NUM_SHARDS -mysql_bind_host ${MYSQL_BIND_HOST:-127.0.0.1} -mysql_server_version 5.7.9-vitess
```

Comment:
It will be nice to allow users to override this with an env variable.
We can default to `5.7.9-Vitess` which is the internal default as well.

Reply:
Done